text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
+
+#### Agency Pages
+* Highlight agencies building websites on Gatsby
+* "Official" landing pages to help agencies in their marketing efforts
+* Allow companies interested in building Gatsby sites to find agencies nearby them
+* Help agencies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
```

Comment:
I'm still not convinced that agency/individual/company pages require a hard separation.

It seems to me that we have three functions in these pages:

1. A one-to-many relationship between an entity (i.e. individual, agency, or company) and Gatsby sites created by it
2. A boolean flag for whether or not this entity is available for hire
3. A one-to-many relationship between an entity and job listings

If we design an "entity" page and create components to surface each of these things, this could effectively be a single template with three top-level components that read from YAML values (e.g. "is there an array of job postings? show the 'available jobs' component").

I bring this up because I can see scenarios where an individual developer has enough work to need a subcontractor, and we don't want to force them to change their URL or create a duplicate post just to gain the ability to recruit from our community.

URL structure could also explicitly signal that this is a community listing, e.g. `https://gatsbyjs.org/community/acme-corp`, `https://gatsbyjs.org/community/jane-q-developer`.

I'm willing to be convinced otherwise, but I worry that arbitrary separation will cause problems as the community expands. Filtering a single set seems like a more scalable solution: it's easier to add an "agency" category and create `https://gatsbyjs.org/community/agencies/` as a subset of all data than to try and subdivide three independent data sets later on.

Reply:
I agree with the idea of a single template with three top-level components, in fact I mention this later :)

I'm torn on the URL schema. I agree that an individual being able to have jobs open would prevent needing to duplicate listings (although that is a bit of an edge case). But it seems like `/people/[person-name] `, `agencies/[agency-name]`, `companies/[company-name]` are more descriptive URLs, and don't want to throw the baby out with the bath water. We have pretty good use cases in mind for each category. 

Also happy with eg `community/people/` to emphasize these are "unofficial" community profiles, though this would be the first three-part slug on the .org site.